test/occupation.jl

```diff
@@ -134,3 +134,28 @@ if mpi_nprocs() == 1 # can't be bothered to convert the tests
     end
 end
 end
+
+if mpi_nprocs() == 1 # can't be bothered to convert the tests
+@testset "Fixed Fermi level" begin
+    testcase = magnesium
+    Ecut = 5
+    fft_size = [15, 15, 15]
+    kgrid  = [2, 3, 4]
+
+    parameters = (
+        (DFTK.Smearing.FermiDirac(), 0.01, 0.2898048739348472, ≈),
+        (DFTK.Smearing.FermiDirac(), 0.01, 0.40, >),
+        (DFTK.Smearing.FermiDirac(), 0.01, 0.20, <),
+    )
+
+    for (smearing, temperature, εF_ref, op) in parameters
+        model = Model(silicon.lattice, testcase.atoms, testcase.positions;
+                      temperature, smearing, terms=[Kinetic()], fixed_fermi_level=εF_ref)
```

Comment:
Why only kinetic?

Reply:
`εF_ref` is a misleading variable name (not a reference but an input argument), just call it `εF` ?